init_scripts/init.sql

```diff
@@ -0,0 +1,56 @@
+--
+-- This init file was generated from a postgreSQL database dump of the
+-- production knack-serivces database in May 2025. It is for basic testing
+-- purposes only and may not reflect the current production state
+--
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+--
+-- Name: api; Type: SCHEMA; Schema: -; Owner: -
+--
+CREATE SCHEMA api;
+
+
+create table api.public_safety_incidents
```

Comment:
Charlie, let's change this to `incidents`. i'm going to make this change in prod. reminder to update this endpoint in the ETL's HTTP requests as well.